core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -614,10 +614,12 @@ var ExplorationEditorMainTab = function() {
       ImageClickInput: 'General',
       ItemSelectionInput: 'General',
       MultipleChoiceInput: 'General',
+      NumericInput: 'General',
       TextInput: 'General',
       FractionInput: 'Math',
       GraphInput: 'Math',
       LogicProof: 'Math',
+      // eslint-disable-next-line no-dupe-keys
       NumericInput: 'Math',
```

Comment:
This doesn't seem right. You can't have duplicate keys in JavaScript objects, can you?

Reply:
Done.